src/SwissTournament.sol

```diff
@@ -34,7 +34,7 @@ abstract contract SwissTournament {
     // indexer value for `matchBook`
     uint256 public matchBookHead;
     uint256 public matchBookTail;
-    uint256 public numPlayers;
+    uint16 public numPlayers;
```

Comment:
any rational for `uint16` here? I mean practically speaking, resolving a 65,000-player tournament would be quite the commotion lol. just curious if theres a optimizooor's rationale since I'm still new to the optimization game

Reply:
ehhh yeah idk why i really did that, it will maybe reduce deployment costs a smidge but nothing substantial. I added the gas report file now.

Reply:
and uint64 gives you 18 quintillion possible player id's, would be hard to collide them. idk the math on collision chance calculation